builder/vsphere/driver/vm.go

```diff
@@ -175,11 +175,21 @@ func (d *Driver) CreateVM(config *CreateConfig) (*VirtualMachine, error) {
 		return nil, err
 	}
 
-	if config.USBController {
-		t := true
-		usb := &types.VirtualUSBController{
-			EhciEnabled: &t,
+	t := true
+	for _, usbType := range config.USBController {
+		var usb types.BaseVirtualDevice
+		switch usbType {
+		// handle "true" and "1" for backwards compatibility
+		case "usb", "true", "1":
```

Comment:
That will work for the cases we put `"true"` or `"1"` but what if we put `1` or `true` as a value here ?

Reply:
Tested locally with JSON and it works with `1` and `true` as well.

Reply:
if you have `"usb_controller": true`, it becomes `"1"` which is why I have that.

Reply:
the question is, should we update so that we handle `"usb_controller": false` as well

Reply:
> the question is, should we update so that we handle "usb_controller": false as well

I think that with Packer if something is possible it's almost always sure someone tried/uses it, so I think yes.

Reply:
`"usb_controller": false` and `"usb_controller": "false"` are handled now

Reply:
Super Nice ! 🎆